db/migrate/20230609140917_create_comment.rb

```diff
@@ -4,7 +4,7 @@ class CreateComment < ActiveRecord::Migration[7.0]
   def change
     create_table :comments do |t|
       t.string :text
-
+      add_reference :comments, :author, column: :author_id, null: false, foreign_key: { to_table: :users }
```

Comment:
- [ ] Kindly move this piece of code outside the `create_table :comments` block, let the code be between `line12 & line13`
This is how our code should like like but not exactly the same
```
create_table :comments do |t|
      t.text :text
      t.integer :author_id
      t.integer :post_id

      t.timestamps
end
add_foreign_key :comments, :users, column: :author_id
add_foreign_key :comments, :posts, column: :post_id
add_index :comments, :author_id
add_index :comments, :post_id
```